### DISCUSSION TITLE
srsUE stuck in 'Attaching UE' status, and never 'Random Access Transmission' nor create 'PDU session'

### DISCUSSION THREAD

**Discussion by initChat** [srsran/srsRAN_Project#445] [General Help]
I follow this tutorial (https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html). Finish install everything.
I ran open5gs docker -> nearRT-RIC -> gNB -> srsUE
But the UE seems to not connect to the networkm, what should i do?

UE display
```
$ sudo ./srsue ./ue_zmq.conf 
Active RF plugins: libsrsran_rf_blade.so libsrsran_rf_zmq.so
Inactive RF plugins: 
Reading configuration file ./ue_zmq.conf...

Built in Release mode using commit ec29b0c1f on branch master.

Opening 1 channels in RF device=zmq with args=tx_port=tcp://127.0.0.1:2001,rx_port=tcp://127.0.0.1:2000,base_srate=23.04e6
Supported RF device list: bladeRF zmq file
CHx base_srate=23.04e6
Current sample rate is 1.92 MHz with a base rate of 23.04 MHz (x12 decimation)
CH0 rx_port=tcp://127.0.0.1:2000
CH0 tx_port=tcp://127.0.0.1:2001
Current sample rate is 23.04 MHz with a base rate of 23.04 MHz (x1 decimation)
Current sample rate is 23.04 MHz with a base rate of 23.04 MHz (x1 decimation)
Waiting PHY to initialize ... done!
Attaching UE...
```
The installed system
- srsRAN_Project for open5gs docker, and gNB
- srsRAN_4G for srsUE
- flexric `br-flexric` for flexRIC and xApp

log of other installation.
[nearRT-ric.txt](https://github.com/srsran/srsRAN_Project/files/14135046/nearRT-ric.txt)
[open5gs_log.txt](https://github.com/srsran/srsRAN_Project/files/14135047/open5gs_log.txt)
[gnb_log.txt](https://github.com/srsran/srsRAN_Project/files/14135045/gnb_log.txt)

**Comment by initChat** [comment]
It is my fault using wrong configuration files

**Reply by qianqian2023** [reply]
Hello, I've also encountered this problem. I also used the ue_zmq.conf file and shouldn't use this file？
